src/components/ContainerDetailsSubheader.react.js

```diff
@@ -54,12 +54,18 @@ var ContainerDetailsSubheader = React.createClass({
     }
     return (this.props.container.State.Downloading || this.props.container.State.Restarting);
   },
-  stopDisabled: function () {
+  disableStop: function () {
     if (!this.props.container) {
       return false;
     }
     return (this.props.container.State.Downloading || this.props.container.State.ExitCode);
   },
+  disableStart: function () {
```

Comment:
@jeffdm tried to keep the name consistent with the other disable methods, should I change the name of everything else?